holmes.py

```diff
@@ -100,6 +100,11 @@ def init_logging(verbose = False):
     "--slack-channel",
     help="Slack channel if --destination=slack (experimental). E.g. #devops",
 )
+opt_dump_raw_output: Optional[str] = typer.Option(
+    None,
+    "-raw-json-file",
```

Comment:
Can we name it --json-output-file please.
If you name the cli argument `json_output_file` then I don't think you'll have to explicitly specify the cli flag.